File: Lista2/Python/Lcasa2/Ex2.py
n = int(input("Digite um número entre 0 e 100: "))
chave = 37
if n > 0 and n < 100:
    if n <= chave:
        print(f"A distância do número {n} para 37 é igual a {chave - n}")
    else:
        print(f"A distância do número {n} para 37 é igual a {n - chave}")
else:
    print("Número inválido")

File: Lista2/C/Lcasa2/Ex2.c
#include <stdio.h> 

int main()
{
	int n, chave;
	printf("Digite um numero entre 0 e 100: ");
	scanf("%i", &n);
	chave = 37;
	if (n > 0 && n < 100) {
		if (n <= chave) {
			printf("A distancia do numero %i para 37 e igual a %i", n, chave-n);
		} else {
			printf("A distancia do numero %i para 37 e igual a %i", n, n - chave);
		}
	} else {
		printf("Numero invalido");
	}
}

